shivakrishna/Assignment2/SWOT.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
+<GridLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:columnCount="2"
+    android:rowCount="4">
+
+    <TextView
+        android:id="@+id/strength"
+        android:layout_width="204dp"
+        android:layout_height="40dp"
+        android:text="STRENGTHS"
+        android:textSize="20sp"
+        android:textStyle="italic"
+        android:layout_column="0"
+        android:layout_row="0"
+        android:background="#005005"/>
+
+    <EditText
+        android:id="@+id/tS"
+        android:layout_width="204dp"
+        android:layout_height="255dp"
```

Comment:
Your layout will break after entering certain number of lines in any EditText view because you are setting a fixed height of 255dp. 
![Imgur](http://i.imgur.com/0AvPQWQm.png)
Let me introduce you to "weights". You can use weight attribute to define the height or width of a view in this case. Since you are using GridLayout, you have two types of weights rowWeight for height and columnWeight for width. Remember when you use weights you have to set the height/weight to 0dp for that particular view. Try adding this in all your EditText views.
`android:layout_height="0dp"
 android:layout_rowWeight="1"`